### PR TITLE
209 add search feature to the header

### LINKED ISSUE #209: Add search feature to the header
a form with a search input that opens the search page with the results and then goes to another page of the website

specs: https://projects.netcentric.biz/wiki/pages/viewpage.action?spaceKey=AICOM&title=Basic+Search+Results+Page+-+CN+Website+Search

### PR DESCRIPTION
**Pull request by joanbeltran**
Fix #209 

Test URLs:
- Before: https://main--cn-website--netcentric.hlx.page/search?terms=test
- After: https://209-add-search-feature-to-the-header--cn-website--netcentric.hlx.page/search?terms=test

### CODE REVIEW

**Comment by joanbeltran** [comment]
The search input has to be included in the header before testing

File: blocks/logos/logos.js [hunk 1]
```diff
@@ -300,23 +300,4 @@ export default function decorate(block) {
       slider.init();
     }
   });
-  const slider = new CarouselSlider(block, {
```
Comment (nc-andreashaller) [inline blocks/logos/logos.js:303]: has this something to do with search?

File: blocks/search-results/search-results.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        this.searchResults = result.data.search;
+        this.showResults();
+      })
+      .catch(() => {
+        this.showResults();
+      });
+  }
+
+  showResults() {
+    let resultsText = window.placeholders?.default?.noResultsFor.replace('{value}', this.searchterm) || `No results for ${this.searchterm}`;
```
Comment (nc-andreashaller) [inline blocks/search-results/search-results.js:50]: not sure if this is a standard but I am used to i18n placeholders like `{}` or `{0}`, should we use the same?

**Review by nc-andreashaller — APPROVED** [review]

**Comment by nc-andreashaller** [comment]
Did you check the search result page with lighthouse?

File: blocks/logos/logos.js (hunk 1 above)
Comment (joanbeltran) [reply blocks/logos/logos.js:303]: no, but It is some code I forgot there and passed a previous PR

**Pull request merged**